src/org/usfirst/frc/team694/robot/subsystems/Drivetrain.java

```diff
@@ -89,6 +97,12 @@ public Drivetrain() {
         differentialDrive = new DifferentialDrive(leftTopMotor, rightTopMotor);
 
         gyro = new ADXRS450_Gyro();
+<<<<<<< HEAD
+=======
+        
+        // Not sure about this next line: (what is kMXP?)
```

Comment:
SPI.kMXP is the port that we grab our accelerometer from. We connect our accelerometer to the Rio's SPI port instead of its I2C or Serial Port